tools/replaceColumn/replaceColumn.xml

```diff
@@ -35,24 +35,23 @@ with open("output_file", 'w') as output:
                 output.write(line)
                 continue
 
-            line_content = line.rstrip().split(delimiter_local)
+            line_content = line.rstrip('\n\r').split(delimiter_local)
 
-            out = list()
-            for j, line_content_column in enumerate(line_content):
-                if j == column:
-
-                    if line_content_column in conversion:
-                        out.append(conversion[line_content_column])
-                    elif unk_strat == "print":
-                        out.append(line_content_column)
-                    elif unk_strat == "error":
-                        raise Exception('ERROR: Encountered a value [%s] in the file that is not in the replacements file and is not commented with [%s]' % (line_content_column, comment_str))
-                else:
-                    out.append(line_content_column)
-
-            if len(out) == len(line_content):
-                output.write('%s\n' % delimiter_local.join(out))
+            if column >= len(line_content):
+                continue
```

Comment:
This is somewhat questionable, but preserves the behavior of the previous version.

Reply:
we could fail instead?

Reply:
Yes, that's the simple alternative. We could also couple the behavior for short lines to the general mode (skip/error/pass), which would also not be too complicated to do.
I was just reluctant to change this cause it may force people to revise WFs when they update to the new tool version.
But if you think it's worth it I can change it to either of the two behaviors above.

Reply:
if we bump the version I think this is fine and afaik the supplied value, if this fails, is invalid

Reply:
Ok, I'll change this. Regarding to what:
I can see use-cases for the more complex implementation where users would want to skip/keep unmodified truncated lines instead of erroring out (like certain lines of the input lacking an annotation so there's nothing to replace, but you don't want to lose these lines either) so I'd prefer being able to handle such situations.